src/app/units/states/edit/directives/unit-details-editor/unit-details-editor.component.html

```diff
@@ -145,11 +148,11 @@ <h3 class="panel-title">Update Unit</h3>
           <div class="col-sm-9">
             <div class="col-sm-4 input-group">
               <input
-                datepicker-popup="{{format}}"
```

Comment:
same problem here. no replacement for deleted functiomnality

Reply:
I have fixed the code, please review again.